modules/sandboxed-containers-deleting-resources.adoc

```diff
@@ -0,0 +1,62 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-resources_{context}"]
+= Deleting {sandboxed-containers-first} resources
+
+You can remove and uninstall the `kata` runtime and all its related resources, such as CRI-O config and `RuntimeClass`, from your cluster.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Delete the `KataConfig` custom resource(CR) by running the following command:
++
+[source,terminal]
+----
+$ oc delete kataconfig <KataConfig_CR_Name>
+----
+
+. Delete the `KataConfig` custom resource definition(CRD) by running the following command:
```

Comment:
@jensfr , @cpmeadors , @zanetworker - Can you 3 review this step along with the verification steps below on this file?